### PR TITLE
Use noise curves from so_noise_models

### PR DESCRIPTION
**Pull request by zonca**
Necessary for https://github.com/simonsobs/map_based_simulations/issues/21

### CODE REVIEW

**Comment by zonca** [comment]
@mhasself I was comparing the old and new noise models.

For now working on the SA, I was expecting no difference between the old and new models, as I thought only the LA atmosphere changed. But I see a difference under 1%:

https://gist.github.com/zonca/7e6a02ef752920d86368f1842c195efc

is this expected?

**Comment by zonca** [comment]
it is a factor of `1.0464` for all ells, the old noise curves have higher noise by that factor

**Comment by mhasself** [comment]
I don't see this discrepancy when comparing old and new models... can you point me to the exact calls issued in the two cases?

It's probably something to do with survey time or area being handled a little differently.

**Comment by zonca** [comment]
thanks @mhasself see the notebook https://gist.github.com/zonca/7e6a02ef752920d86368f1842c195efc

**Comment by mhasself** [comment]
It looks like you specify SA_years_LF=1 in the old model.  The reference code plotting example had that parameter as 2, which became the default configuration in the SA model in v3.1.1.  That doesn't explain everything... but does it explain anything?

**Comment by zonca** [comment]
yes @mhasself I had all the tubes configuration wrong, now I get a ratio of:

1.00068493 at 225 GHz 1.00068493 at 27 GHz

I'll keep checking the whole configuration for other errors.

**Comment by zonca** [comment]
got it! in the new simulator, a year is 365.25, in the old one 365. 
now I get the same result.

image

**Comment by zonca** [comment]
@msyriac please review this pull request.
Next I'll prepare another pull request with:
* read new hitmaps
* generate noise spectra with correlations

at that point I'll also update the docs. For now I only have the docstrings updated but the docs are obsolete.

File: mapsims/data/example_config.toml [hunk 1]
```diff
@@ -21,7 +21,10 @@ num = 0
     LA_number_LF = 1
     LA_number_MF = 4
     LA_number_UHF = 2
-    SA_years_LF = 1
+    SA_years = 4.99657768652
```
Comment (msyriac) [inline mapsims/data/example_config.toml]: Can we add a comment in the toml that this number is due to the leap year correction?

File: mapsims/tests/test_runner.py [hunk 2]
```diff
@@ -37,7 +37,10 @@ def test_from_classes():
         LA_number_LF=1,
         LA_number_MF=4,
         LA_number_UHF=2,
-        SA_years_LF=1,
+        SA_years=365 * 5 / 365.25,
```
Comment (msyriac) [inline mapsims/tests/test_runner.py]: Same here -- add a comment

**Comment by msyriac** [comment]
Ok, done with review, just needed some descriptive comments. For "read new hitmaps", do you mean you'll be adding new hitmaps with updated scan strategy? See the versioning I've done in the directories for this (with a version argument in the noise class). Also let me know when you add them, so I can convert to CAR.

**Pull request merged**